app/services/staff_nomis_checker_factory.rb

```diff
@@ -0,0 +1,10 @@
+class StaffNomisCheckerFactory
+  def self.for(visit)
+    case visit.processing_state
+    when 'requested'
```

Comment:
Agreed, was surprised we didn't have them. Didn't want to introduce them as part of this PR because it would have introduced a lot of changes.